jina/docker/hubapi.py

```diff
@@ -117,6 +117,46 @@ def _list(logger, image_name: str = None, image_kind: str = None,
         return manifests
 
 
+def _docker_auth(logger) -> Optional[Dict[str, str]]:
+    """ Use Hub api to get docker creds
+
+    :return: a dict of specifying username and password
+    """
+    with resource_stream('jina', '/'.join(('resources', 'hubapi.yml'))) as fp:
+        hubapi_yml = JAML.load(fp)
+        hubapi_url = hubapi_yml['hubapi']['url'] + hubapi_yml['hubapi']['docker_auth']
+
+    access_token = _fetch_access_token(logger)
+    if not access_token:
+        logger.error(f'user has not logged in. please login using command: {colored("jina hub login", attrs=["bold"])}')
+        return
+
+    headers = {
+        'Accept': 'application/json',
+        'authorizationToken': access_token
+    }
+
+    try:
+        import requests
+        response = requests.get(url=f'{hubapi_url}',
+                                 headers=headers)
+        if response.status_code == requests.codes.ok:
+            import json
+            import base64
+            json_response = json.loads(response.text)
+            encoded_username = json_response['docker_username']
+            encoded_password = json_response['docker_password']
+            decoded_username = base64.b64decode(encoded_username).decode('ascii')
+            decoded_password = base64.b64decode(encoded_password).decode('ascii')
+            docker_creds = {'docker_username': decoded_username, 'docker_password': decoded_password} 
+            logger.info(f'Successfully fetched docker creds for user')
```

Comment:
This can be changed to `logger.debug` as user is not aware that we are fetching the docker creds for them.